pkg/cleanup/cleanup.go

```diff
@@ -0,0 +1,235 @@
+package cleanup
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/icinga/icingadb/pkg/icingadb"
+	"github.com/icinga/icingadb/pkg/utils"
+	"go.uber.org/zap"
+	"sync"
+	"time"
+)
+
+// Tables type defines mapping of history tables with corresponding period.
+type Tables map[string]string
```

Comment:
Rename to `HistoryRetention` and use map[string]uint. Uint for days. time.Duration does not really make sense. Make sure to adjust all other code and docs as well.